src/interfaces/IOptionSettlementEngine.sol

```diff
@@ -267,7 +269,7 @@ interface IOptionSettlementEngine {
     /// @dev Represents the total amount of options written and exercised for a group of
     /// claims bucketed by day. Used in fair assignement to calculate the ratio of
     /// underlying to exercise assets to be transferred to claimants.
-    struct OptionLotClaimBucket {
+    struct OptionsBucket {
```

Comment:
I think we lose some meaning with this name, but I don't like the previous name I put either. Could we take one more pass at it?